TYPO3.Neos/Resources/Private/TypoScript/Prototypes/ImageUri.ts2

```diff
@@ -2,7 +2,9 @@
 prototype(TYPO3.Neos:ImageUri) {
 	@class = 'TYPO3\\Neos\\TypoScript\\ImageUriImplementation'
 	maximumWidth = 2560
+	width = NULL
 	maximumHeight = 2560
```

Comment:
I think this magic number should be set to NULL in this case the image w/h will be used automatically